cmd/backport-audit/main.go

```
@@ -0,0 +1,991 @@
+	case statusPending:
+		return cfg.yellow
+	case statusDropped:
+		return cfg.cyan
+	case statusBackported, statusInBranch:
+		return cfg.green
+	}
+	return ""
+}
+
+func capture(name string, args ...string) (string, error) {
+	out, err := exec.Command(name, args...).Output()
```

golangci-lint (2.13.2)

[error] os/exec.Command must not be called. use os/exec.CommandContext

(noctx)